script/upgrades/eXOF/Config.sol

```diff
@@ -60,32 +62,37 @@ library eXOFConfig {
       enabled: true,
       threshold: FixidityLib.newFixedFraction(3, 100), // 0.03
       cooldown: 30 minutes,
-      smoothingFactor: 0
+      smoothingFactor: 1e24 // 1
     });
-    rateFeedConfig.dependentRateFeeds = Arrays.addresses(contracts.dependency("EURXOFRateFeedAddr"));
+    rateFeedConfig.dependentRateFeeds = Arrays.addresses(Config.rateFeedID("EURXOF"));
   }
 
   /**
    * @dev Returns the configuration for the EURXOF rate feed.
    */
-  function EURXOF_RateFeedConfig(
+  function EUROCXOF_RateFeedConfig(
     Contracts.Cache storage contracts
   ) internal returns (Config.RateFeed memory rateFeedConfig) {
-    rateFeedConfig.rateFeedID = contracts.dependency("EURXOFRateFeedAddr");
+    rateFeedConfig.rateFeedID = Config.rateFeedID("EUROCXOF");
     rateFeedConfig.valueDeltaBreaker0 = Config.ValueDeltaBreaker({
       enabled: true,
-      threshold: FixidityLib.newFixedFraction(5, 1000), // 0.005
```

Comment:
Removing this value breaker on EURXOF has two consequences:
1. Only one of the base pairs that build EUROC/XOF have a tight (50bps) value breaker
2. The CELO/eXOF pool will only inherit the 10% value breaker. In particular, CELO/eXOF will stay active while EUROC/eXOF is suspended until it auto-recovers.

I think we could live with 1, by maybe lowering EUROC/XOF value breaker threshold to 50bps. 
But 2 does not feel great. Allowing minting / redemption through CELO/eXOF while we do not via EUROC